tenzir/integration/tests.yaml

```diff
@@ -1129,6 +1129,10 @@ tests:
         input: data/cef/checkpoint.log
       - command: exec 'read lines -s | summarize n=count(.) | write json -c'
         input: data/cef/checkpoint.log
+      - command: exec 'read json | write lines'
+        input: data/json/all-types.json
+      - command: exec 'read json | put e | write lines'
+        input: data/json/all-types.json
```

Comment:
cc @lava: new tests in the old format.

Reply:
Can we please rewrite this as a BATS tests? Otherwise there's a danger that these new tests will get accidentally swallowed, since the BATS pr is going to remove the whole `tenzir/integration` directory.